twitter.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Project</title>
+    <link rel="stylesheet" href="twitter1.css" />
+  </head>
+  <body>
+    <!-- ... icon -->
+    <!-- <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
I think you might have pushed the work you had done when Mike told you to create the PR - but just in case, it's always nice to remove any commented code we no longer need. Just to keep the code base clean and readable :)